feat: update model to add type restrictions (#233)
openfga/openfga.dev: docs/content/modeling/advanced/github.mdx

            this: {}, // the userset of all users related to the repo as "member"
          },
        },
        metadata: {
          relations: {
            member: { directly_related_user_types: [{ type: 'user' }, { type: 'team', relation: 'member' }] },
          },
        },
      },
    ],
  }}
/>

> In addition, the repo's relations should be updated to allow team's member as directly related user types.

Reviewer: ```suggestion
In addition, the repo's relations should be add team member as a directly related user types.
```
